source/_posts/2023-09-06-release-20239.markdown

```diff
@@ -15,39 +15,27 @@ og_image: /images/blog/2023-09/social.png
 
 <a href='/integrations/#version/2023.9'><img src='/images/blog/2023-09/social.png' style='border: 0;box-shadow: none;'></a>
 
-<!-- BELOW NEEDS TO BE REMOVED BEFORE THE RELEASE -->
-
-These are the beta release notes for Home Assistant Core 2023.9 (and is thus a
-work in progress).
+Home Assistant Core 2023.9! 🎉
 
-If you encounter any issues with the beta release, please report them on GitHub:
+Frenck is enjoying a well earned holiday, so I'm taking care of the release this month.
 
-- Issues with integrations, automations and such (Core related):<br>
-  <https://github.com/home-assistant/core/issues>
-- Issues with the frontend/UI/dashboards:<br>
-  <https://github.com/home-assistant/frontend/issues>
-- Issues with the documentation:<br>
-  <https://github.com/home-assistant/home-assistant.io/issues>
+This month on the 17th it has been 10 years since Home Assistant started. We'll be publishing a video to the [Home Assistant YouTube channel](https://www.youtube.com/channel/UCbX3YkedQunLt7EQAdVxh7w) to celebrate, reflect and look ahead. We hope you can join us on the 17th @ 12:00 PDT / 21:00 CET!
```

Comment:
```suggestion
This month, on the 17th, it has been 10 years since Home Assistant started. We'll be publishing a video to the [Home Assistant YouTube channel](https://www.youtube.com/channel/UCbX3YkedQunLt7EQAdVxh7w) to celebrate, reflect and look ahead. We hope you can join us on the 17th @ 12:00 PDT / 21:00 CET!
```